config/hubs/2i2c.cluster.yaml

```diff
@@ -48,10 +65,8 @@ hubs:
           config:
             Authenticator:
               allowed_users: &staging_users
-                - yuvipanda@gmail.com
-                - colliand@gmail.com
-                - choldgraf@gmail.com
-                - georgiana.dolocan@gmail.com
+                - *staff-google-oauth
+                - anextrausername
```

Comment:
this is just to show off what I'm talking about here - obviously we don't want to actually add `anextrausername` to our hub :-)